docs/content/docs/data-structures/tree.md

```diff
@@ -547,7 +545,7 @@ on<K extends keyof TreeChangeEvents>(
 	): () => void;
 ```
 
-`Tree.on` assigns the specified `listener` function to the specified `eventType` for the specified `node`. The `node` can be any node of the tree. The `eventType` can be either "treeChanged" or "nodeChanged". `nodeChanged` fires whenever the given node changes. `treeChanged` fires whenever the given node or any of the nodes in its child subtree changes.
+`Tree.on` assigns the specified `listener` function to the specified `eventType` for the specified `node`. The `node` can be any node of the tree. The `eventType` can be either "treeChanged" or "nodeChanged". `nodeChanged` fires whenever the given node changes (values of the fields in the node). `treeChanged` fires whenever the given node or any of the nodes in its child subtree changes.
```

Comment:
I'm not sure I understand the new contents in the parenthesis. If the goal is to differentiate `nodeChanged`'s behavior from that of `treeChanged`, maybe just saying something like "i.e., when the values of any of the node's fields change. This does not include changes made to child nodes' fields." @alexvy86 you've been working in this space; what do you think?

Reply:
Yeah, it's a finicky space for wording. I'd recommend this. Also replaces `eventType` with `eventName` to match the signature shown above.
```suggestion
`Tree.on` assigns the specified `listener` function to the specified `eventName` for the specified `node`.
The `node` can be any node of the tree.
The `eventName` can be either "treeChanged" or "nodeChanged".
`nodeChanged` fires whenever one or more properties of the specified node change.
`treeChanged` fires whenever one or more properties of the specified node or any node in its subtree, change.
We recommend looking at the documentation of each of the events for more details.
```